Python/RanjanPaudel/task3/mysql_handler.py

```diff
@@ -0,0 +1,25 @@
+import sqlalchemy
+from sqlalchemy.engine import Connection
+from sqlalchemy import Table
+
+import mysql_db.config as config
+import mysql_db.models as mysql_models
+
+tables = mysql_models.meta_data.tables
+conn = Connection(config.engine)
+
+if __name__ == "mysql_handler":
+    def insert_many(table_name, dict_list):
+        with conn.begin():
```

Comment:
Try to implement using context manager. 
with connection.cursor() as cursor:
cursor.execute(sql query)